### DISCUSSION TITLE
Contribute to cdnjs.

### DISCUSSION THREAD

**Discussion by baraichgurvir** [cdnjs/cdnjs#14045] [General] [answered]
Hi there,
Hey developers! I wanted to add a file to cdnjs. This file is a library which creates a form, can listen for callbacks, prevents get data to be displayed on the URL bar. I hove you all will consider this library useful.

I saw the cdnjs/packages repo but, I am confused how do I contribute to cdnjs. if I could get some information about the process it could be very helpful for me.

Thanks

**Comment by MattIPv4** [comment, marked as answer]
👋 Please have a read through the [CONTRIBUTING.md](https://github.com/cdnjs/packages/blob/master/CONTRIBUTING.md) file that we have in the cdnjs/packages repo, it covers how to configure auto-update for a library. You can also look at existing package JSON files in that repository for live examples of what is expected.
